docker-compose.yaml

```diff
@@ -0,0 +1,65 @@
+version: '2'
+services:
+  app:
+    build:
+      context: .
+    volumes:
+      - .:/app
+      - /app/node_modules
+    working_dir: /app
+    environment:
+      - SELENIUM_URL=http://selenium:4444/wd/hub
```

Comment:
`docker-compose` launches 4 containers: `app`, `selenium`, `firefox`, `chrome`.
* `app` runs tests.
* Protractor on `app` connects to `selenium` to launch web browsers and control them.
* `firefox` and `chrome` registers themselves to `selenium`. `selenium` connects to `firefox` or `chromes` based on the capability specified by protractor.